.github/workflows/deploy.yml

```diff
@@ -27,17 +27,33 @@ jobs:
   deploy:
     name: Deploy
     runs-on: ubuntu-latest
+    container:
+        image: ubuntu:latest
```

Comment:
This seems like a good idea and it is nice to not be depending on a moving target.

The only downside is that we would need to put in a small amount of extra effort to make sure that we stay up to date with a fairly recent version of Ubuntu over time (ideally, an LTS version). However, this seems like a pretty minor point. Using 22.04 should be fine for a while since a new LTS version probably won't be released until the April 2024 timeframe according to the [Ubuntu release cycle](https://ubuntu.com/about/release-cycle).